src/web/client/test/integration/portalSchemaReader.test.ts

```diff
@@ -0,0 +1,225 @@
+            multiEntityURL: "{dataverseOrgUrl}/{api}/{data}/{version}/{entity}",
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } as any;
+
+        const result = getDataSourcePropertiesMap("test");
+
+        //Assert
+        for (const key in dataSourceProperties) {
+            expect(result.get(key)).eq(dataSourceProperties[key]);
+        }
+    });
+
+    it("getDataSourcePropertiesMap_withEmptySchema_shouldBeEmptyMap", () => {
+        //Act
+        stub(schemaHelperUtil, "getPortalSchema").returns(
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            portalSchemaReadeMock.schema_data_with_empty_dataSourceProperties as any
+        );
+
+        //Action
+        const result = getDataSourcePropertiesMap("testSchema");
+
+        //Assert
+        expect(result.size).eq(0);
+    });
+
+    it("getEntitiesFolderNameMap_withValidEntitiesSchema_shouldBeMap", () => {
+        //Act
+        const folderNameMap = new Map<string, string>([
+            [schemaEntityKey.FILE_FOLDER_NAME, "testFolderName"],
+        ]);
+
+        const folderNameMap1 = new Map<string, string>([
+            [schemaEntityKey.FILE_FOLDER_NAME, "testFolderName1"],
+        ]);
+
+        const folderNameMap2 = new Map<string, string>([
+            [schemaEntityKey.FILE_FOLDER_NAME, "testFolderName2"],
+        ]);
+
+        const entitiesSchemaMap = new Map<string, Map<string, string>>([
+            ["webfiles", folderNameMap],
+            ["webpages", folderNameMap1],
+            ["webtemplates", folderNameMap2],
+        ]);
+
+        //Action
+        const result = getEntitiesFolderNameMap(entitiesSchemaMap);
+
+        //Assert
+        expect(result?.get("webfiles")).equal("testFolderName");
+        expect(result?.get("webpages")).equal("testFolderName1");
+        expect(result?.get("webtemplates")).equal("testFolderName2");
+    });
+
+    it("getEntitiesFolderNameMap_With_Different_schemaEntityKey_Should_Not_Map", () => {
+        //Act
+        const folderNameMap = new Map<string, string>([
+            [schemaEntityKey.ATTRIBUTES, "testFolderName"],
+        ]);
+
+        const folderNameMap1 = new Map<string, string>([
+            [schemaEntityKey.DATAVERSE_ENTITY_NAME, "testFolderName1"],
+        ]);
+
+        const folderNameMap2 = new Map<string, string>([
+            [schemaEntityKey.FETCH_QUERY_PARAMETERS, "testFolderName2"],
+        ]);
+
+        const entitiesSchemaMap = new Map<string, Map<string, string>>([
+            ["webfiles", folderNameMap],
+            ["webpages", folderNameMap1],
+            ["webtemplates", folderNameMap2],
+        ]);
+
+        //Action
+        const result = getEntitiesFolderNameMap(entitiesSchemaMap);
+
+        //Assert
+        expect(result.size).eq(0);
+    });
+
+    it("getEntitiesFolderNameMap_With_Different_entitiesSchemaKey_Should_Not_Map", () => {
+        //Act
```

Comment:
What is the difference between above and this test case

Reply:
In the above one i am sending the "schemaEntityKey.FILE_FOLDER_NAME" as key that will be map in the actual function but in the below test case i am not sending the key as "schemaEntityKey.FILE_FOLDER_NAME" so it wont map the key and will return empty Map

Reply:
The naming convention is still not correct

Reply:
Done and changes pushed changes